beacon_node/beacon_chain/src/data_availability_checker/overflow_lru_cache.rs

```diff
@@ -238,6 +331,17 @@ impl<E: EthSpec> PendingComponents<E> {
             block,
             blobs: Some(verified_blobs),
             blobs_available_timestamp,
+            // TODO(das) Do we need a check here for number of expected custody columns?
```

Comment:
This check has already been done in `is_available` so this is redundant here. I'll remove the TODO